src/sas/qtgui/MainWindow/DataExplorer.py

```diff
@@ -183,8 +183,8 @@ def enableGraphCombo(self, combo_text):
         """
         Enables/disables "Assign Plot" elements
         """
-        self.cbgraph.setEnabled(len(PlotHelper.currentPlots()) > 0)
-        self.cmdAppend.setEnabled(len(PlotHelper.currentPlots()) > 0)
+        self.cbgraph.setEnabled(len(PlotHelper.currentPlotIds()) > 0)
+        self.cmdAppend.setEnabled(len(PlotHelper.currentPlotIds()) > 0)
```

Comment:
```suggestion
        self.cbgraph.setEnabled(PlotHelper.currentPlotIds())
        self.cmdAppend.setEnabled(PlotHelper.currentPlotIds())
```

Possibly more Pythonic to directly use the truthiness

Reply:
I get what your saying, if it was a direct if statement yes, as that would implicity run bool on it, but in this case we would be passing the list of ids to setEnabled, rather than evaluating it as a boolean. I don't think bool(PlotHelper...) is better